social_django/context_processors.py

```diff
@@ -45,11 +46,15 @@ def backends(request):
 
 def login_redirect(request):
     """Load current redirect to context."""
-    value = (
-        request.method == "POST"
-        and request.POST.get(REDIRECT_FIELD_NAME)
-        or request.GET.get(REDIRECT_FIELD_NAME)
-    )
+    try:
+        value = (
+            request.method == "POST"
+            and request.POST.get(REDIRECT_FIELD_NAME)
+            or request.GET.get(REDIRECT_FIELD_NAME)
```

Comment:
When touching this code, it might be a good idea to use ternary operator instead:

```suggestion
            request.POST.get(REDIRECT_FIELD_NAME)
            if request.method == "POST"
            else request.GET.get(REDIRECT_FIELD_NAME)
```